0-common/schema-ddl/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/Linter.scala

```diff
@@ -351,6 +352,28 @@ object Linter {
       }
   }
 
+  final case object duplicateFieldNamesAfterNormalization extends Linter { self =>
```

Comment:
Let's leave it just as `duplicateProperties` as users sometimes have to remember these linters and write them manually.

Reply:
`duplicateProperties` implies that we are checking the duplicate properties however right now, we are only checking duplicate properties after normalization. Should I resolve issue #343 too ?